cryptol-remote-api/src/CryptolServer/Utils.hs

```diff
@@ -0,0 +1,51 @@
+module CryptolServer.Utils where
+
+import Data.Text(Text)
+import qualified Data.Text as Text
+import qualified Data.ByteString as BS
+import Data.Word
+import Data.Bits
+
+import Data.Aeson
+
+newtype BS = BS BS.ByteString
```

Comment:
What is this type? Why do we need it separately from `ByteString`? Including some Haddocks would be helpful.